metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -10,6 +10,10 @@ You must configure queries for the Windows performance counters that you wish
 to collect. The example below collects processor time and disk writes.
 With `format` you can set the output format for a specific counter. Possible values are
 `float` and `long`. If nothing is selected the default value is `float`.
+There is also an optional/required option `instance_name`.
```

Comment:
Saying " optional/required option" seems like a contradiction. I'd reword this section to say something like the following (I've fixed a few grammatical issues):

With `instance_name`, you can specify the name of the instance. Use this setting when: 

* You want to specify an instance name that is different from the computed name. For example, `Total` instead of `_Total`.
* You specify a counter that has no instance. For example, `\TCPIP Performance Diagnostics\IPv4 NBLs/sec indicated without prevalidation`.

I'm not sure what you mean by "For wildcard queries this option is needles." Do you mean that the setting has no effect?